docs/getting-started/1201-ci-environment.md

```diff
@@ -389,5 +390,23 @@ spec:
 
 ```
 
+</TabItem>
+
+<TabItem value="azure-pipelines">
+
+Azure Pipelines does not currently have a native task.
+
+Here is an example of how to still run Dagger in an Azure Pipelines on a Mac or Linux hosted agent.
```

Comment:
A quick suggestion to improve this sentence.

```
Azure Pipelines do not currently supports native task, however it is still possible to run Dagger on Mac, Linux or Windows hosted agent.

To use Dagger on Mac or Linux hosted agent you can copy this file

<file>

Since you cannot use curl on Windows hosted agent, you will need to update the install step to:
```

```yaml
  - task: ChocolateyCommand@0
     inputs:
       command: 'install'
       installPackageId: 'dagger'
       installPackageVersion: '$(DAGGER_VERSION)'
     displayName: "Install Dagger $(DAGGER_VERSION)"
```

Reply:
I have updated the wording to be closer to your suggestion and removed the second Windows only file and replaced it with the install snippit as suggested.